### PR TITLE
Fixed #34678 -- Added Django Forum and Discord server to README.rst.

### PR DESCRIPTION
**Pull request by anorthall**
https://code.djangoproject.com/ticket/34678

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,11 +29,14 @@ ticket here: https://code.djangoproject.com/newticket
...
+* Join the `Django Discord <https://discord.gg/xcRH6mN4fa>`_. Lots of helpful
+  people hang out there.
 
-* Join the django-users mailing list, or read the archives, at
-  https://groups.google.com/group/django-users.
+* Post on the `Django forum <https://forum.djangoproject.com/>`_ with a
+  description of your problem or query.
+
+* Visit the `Django community <https://www.djangoproject.com/community/>`_
+  page for details of more ways to get in touch with other Django users.
```
Comment (felixxm) [inline README.rst:39]: We could add a forum, but I don't see a reason to remove the IRC channel.

Comment (anorthall) [reply README.rst]: Thanks for your feedback! Changed.

File: README.rst [hunk 2]
```diff
@@ -30,11 +30,14 @@ ticket here: https://code.djangoproject.com/newticket
...
 
 * Join the ``#django`` channel on ``irc.libera.chat``. Lots of helpful people
-  hang out there. See https://web.libera.chat if you're new to IRC.
+  hang out there. `Webchat is available <https://web.libera.chat/#django>`_.
 
 * Join the django-users mailing list, or read the archives, at
   https://groups.google.com/group/django-users.
 
+* We've also got the `Django Discord <https://discord.gg/xcRH6mN4fa>`_ and the
+  `Django forum <https://forum.djangoproject.com/>`_.
```
Comment (felixxm) [inline README.rst:39]: _"to get more help: We've also got"_ doesn't sound good to me, maybe
```suggestion
* Join the `Django Discord Community <https://discord.gg/xcRH6mN4fa>`_.

* Join the community on the `Django Forum <https://forum.djangoproject.com/>`_.
```

**Comment by felixxm** [comment]
@anorthall Thanks :+1:

**Pull request merged**